non-oss/pharos_pro/addons/kontena-lens/addon.rb

```diff
@@ -3,6 +3,7 @@
 Pharos.addon 'kontena-lens' do
   version '1.1.0'
   license 'Kontena License'
+  priority 10
```

Comment:
What is the default priority?

Reply:
priority method returns `@priority.to_i`, by default it is 0 (`nil.to_i`). Then the order is the order where addons are introduced in `cluster.yml`